### PR TITLE
API avaliability

### PR DESCRIPTION
**Pull request by abhishekpratapa**
- added api Availability for 10.15

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -193,6 +193,11 @@ if(APPLE)
     # CoreML MLCustomModel requires macOS 10.14 or higher
     set(HAS_MLCUSTOMMODEL TRUE)
   endif()
+
+   # MPSForward Loss is only present on macOS 10.15 or higher.
```
Comment (nickjong) [inline CMakeLists.txt]: I don't think we need this comment, as the new preprocessor symbol is not specific to `MPSForwardLoss`

File: src/ml/neural_net/style_transfer/mps_style_transfer.h [hunk 2]
```diff
@@ -13,6 +13,8 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+#ifdef DHAS_MACOS_10_15
```
Comment (nickjong) [inline src/ml/neural_net/style_transfer/mps_style_transfer.h]: Should be `HAS_MACOS_10_15`, no `D` at the front

File: src/ml/neural_net/style_transfer/mps_style_transfer.h [hunk 3]
```diff
@@ -33,4 +35,6 @@ API_AVAILABLE(macos(10.15))
 
 @end
 
+#endif
```
Comment (nickjong) [inline src/ml/neural_net/style_transfer/mps_style_transfer.h]: Nit: let's comment with ```// #ifdef HAS_MACOS_10_15```

Comment (abhishekpratapa) [reply src/ml/neural_net/style_transfer/mps_style_transfer.h]: Solid

File: src/ml/neural_net/style_transfer/mps_style_transfer.h (hunk 2 above)
Comment (abhishekpratapa) [reply src/ml/neural_net/style_transfer/mps_style_transfer.h]: Done!

File: CMakeLists.txt (hunk 1 above)
Comment (abhishekpratapa) [reply CMakeLists.txt]: Removed comment!

**Pull request merged**